app/assets/stylesheets/components/_task-list.scss

```diff
@@ -104,15 +103,18 @@ $top-border: solid 2px $grey-3;
     border-bottom: solid $stroke-width $grey-2;
   }
 
+  // scss-lint:disable DuplicateProperty
   .gem-c-task-list__step:last-child {
     &:after {
+      height: -webkit-calc(100% - #{$gutter-half}); // fix for iphone 4s
```

Comment:
This is more than just iPhone 4s right? Could we just say 'fallback'?

Could we do the disabling only on the specific line.

Reply:
HOW DO YOU SPOT THESE PRS SO QUICKLY

Reply:
Sure, will change.